Avoids installation into /usr/libexec directory.

Solaris specific. Not suitable for upstream.

--- qt-everywhere-src-6.8.0/qtbase/cmake/QtBuildPathsHelpers.cmake
+++ qt-everywhere-src-6.8.0/qtbase/cmake/QtBuildPathsHelpers.cmake
@@ -172,7 +172,7 @@ endif()
     if (WIN32)
         set(_default_libexec "${INSTALL_ARCHDATADIR}/bin")
     else()
-        set(_default_libexec "${INSTALL_ARCHDATADIR}/libexec")
+        set(_default_libexec "${INSTALL_ARCHDATADIR}/lib/qt6")
     endif()
 
     qt_configure_process_path(
